dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -258,6 +270,10 @@ def main(args):
         for (w1, w2), nb in total_nbytes.items()
     }
 
+    broadcast = (
+        "false" if args.shuffle_join else ("true" if args.broadcast_join else "default")
```

Comment:
```suggestion
        False if args.shuffle_join else (True if args.broadcast_join else "default")
```

Reply:
Unless I'm misunderstanding the reason for using strings here, this will ensure the summary at the end is standardized, we have also `rmm-pool` that is using actual booleans. If there's a reason to use string, I'd prefer `"False"` and `"True"`, so as to mimic Python's conversion to string and thus still make the output look alike.